CHANGELOG/CHANGELOG-2.0.md

```diff
@@ -0,0 +1,89 @@
+  - Remove a few deprecated options including `enableIPSecTunnel`,`multicastInterfaces`, `multicluster.enable` and `legacyCRDMirroring`. ([#5158](https://github.com/antrea-io/antrea/pull/5158), [@luolanzone])
+  - Clean up unused codes for NodePortLocal and remove the deprecated `nplPortRange` config. ([#5943](https://github.com/antrea-io/antrea/pull/6943), [@luolanzone])
+  - Clean up deprecated APIServices. ([#6002](https://github.com/antrea-io/antrea/pull/6002), [@tnqn])
+- Documents are updated to reflect the latest changes with clearer guidance.
+  - Update the OVS pipeline document and workflow diagram to keep them up to date. ([#5412](https://github.com/antrea-io/antrea/pull/5412), [@hongliangl])
+  - Clarify documentation for `IPPool` and `ExternalIPPool` CRDs. ([#6183](https://github.com/antrea-io/antrea/pull/6183), [@antoninbas])
+  - Document Pods using FQDN based policies must respect DNS TTL. ([#6230](https://github.com/antrea-io/antrea/pull/6230), [@tnqn])
+  - Document the limitations of Audit Logging for policy rules. ([#6225](https://github.com/antrea-io/antrea/pull/6225), [@antoninbas])
+- Optimizing Antrea binaries size.
+  - Optimize package organization to reduce antctl binary size. ([#6037](https://github.com/antrea-io/antrea/pull/6037), [@tnqn])
+  - Reduce Antrea cni binary size by removing unnecessary import packages. ([#6038](https://github.com/antrea-io/antrea/pull/6038), [@tnqn])
+  - Strip all debug symbols from Go binaries by default. ([#6035](https://github.com/antrea-io/antrea/pull/6035), [@antoninbas])
+  - Disable cgo for all Antrea binaries. ([#5988](https://github.com/antrea-io/antrea/pull/5988), [@antoninbas])
+- Increase the minimum supported Kubernetes version to v1.19. ([#6089](https://github.com/antrea-io/antrea/pull/6089), [@hjiajing])
+- Add OVS groups dump information to support bundle to help troubleshooting. ([#6195](https://github.com/antrea-io/antrea/pull/6195), [@shikharish])
+- Add `egressNodeName` in flow records for Antrea Flow Aggregator. ([#6012](https://github.com/antrea-io/antrea/pull/6012), [@Atish-iaf])
+- Add `EgressNode` field in the Traceflow Egress observation. ([#5949](https://github.com/antrea-io/antrea/pull/5949), [@Atish-iaf])
+- Upgrade `IPPool` CRD to v1beta1 and make the subnet definition consistent with the one in `ExternalIPPool` CRD. ([#6036](https://github.com/antrea-io/antrea/pull/6036), [@mengdie-song])
+- Request basic memory for antrea-controller to improve its scheduling and reduce its OOM adjustment score, enhancing overall robustness. ([#6233](https://github.com/antrea-io/antrea/pull/6233), [@tnqn])
+- Increase default rate limit of antrea-controller to improve performance for batch requests. ([#6231](https://github.com/antrea-io/antrea/pull/6231), [@tnqn])
+- Remove Docker Support for antrea-agent on Windows. ([#6019](https://github.com/antrea-io/antrea/pull/6019), [@XinShuYang])
```

Comment:
s/Support/support

maybe link to the documentation PR (#6255) as well?

Reply:
Updated and add the link both 6255 and 6202 in the release log.